src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
+import { useContext } from "react";
+import { CustomContext } from "../helpers/Context";
+
+function Header() {
+  const { isTheme = true, toggleTheme } = useContext(CustomContext);
+  return (
+    <>
+      {isTheme ? (
```

Comment:
Why `isTheme` is a boolean? Light theme is considered as non-theme? =)

Reply:
done